frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -541,14 +542,23 @@ common_settings.document = {
 local metadata_folder_str = {
     ["doc"] = _("book folder"),
     ["dir"] = "koreader/docsettings/",
+    ["hash"] = "koreader/hashdocsettings/"
 }
 
 local metadata_folder_help_text = _([[
 Book view settings, reading progress, highlights, bookmarks and notes (collectively known as metadata) are stored in a separate folder named <book-filename>.sdr (".sdr" meaning "sidecar").
 
 You can decide between two locations where these will be saved:
-- alongside the book file itself (the long time default): these sdr folders will be visible when you browse your library directories with another file browser or from your computer, which may clutter your vision of your library. But this allows you to move them along when you reorganize your library, and also survives any renaming of parent directories. Also, if you perform directory synchronization or backups, your settings will be part of them.
-- all inside koreader/docsettings/: these sdr folders will only be visible and used by KOReader, and won't clutter your vision of your library directories with another file browser or from your computer. But any reorganisation of your library (directories or filename moves and renamings) may result in KOReader not finding your previous settings for these books. These settings won't be part of any synchronization or backups of your library.]])
+- alongside the book file itself (the long time default): sdr folders will be visible when you browse your library directories with another file browser or from your computer, which may clutter your vision of your library. But this allows you to move them along when you reorganize your library, and also survives any renaming of parent directories. Also, if you perform directory synchronization or backups, your settings will be part of them.
+- all in koreader/docsettings/: sdr folders will only be visible and used by KOReader, and won't clutter your vision of your library directories with another file browser or from your computer. But any reorganisation of your library (directories or filename moves and renamings) may result in KOReader not finding your previous settings for these books. These settings won't be part of any synchronization or backups of your library.
+- all inside koreader/hashdocsettings/ as hashes: sdr folders are identified not by filepath/filename but by partial MD5 hash, allowing you to rename, move, and copy documents outside of KOReader without sdr folder clutter while keeping them linked to their metadata. However, any file modifications such as writing highlights into PDFs or downloading from Calibre may change the hash, and thus lose their linked metadata. Calculating file hashes may also slow down file browser navigation. This option may suit users with multiple copies of documents across different devices and directories.]])
```

Comment:
Is the "these" really bothersome enough to propagate to all the translators? :-)

Reply:
Was 'this' changed to 'these' in this multiline string? I can change it back if so, was trying to shorten things since it's long but a few chars makes little difference lol

Reply:
Note that this applies to all of these strings. They were all changed in a minuscule manner that I'd prefer not to bother the translators with unless absolutely necessary. ^_^

Reply:
@Frenzie The second one needs to be changed anyway if we're dynamically putting in the path. I'd vote to just propagate the update since there will be other changes too. But if you want I'll still revert the first two bullet points to their original form (excluding the dynamic path), just keep in mind the translators have to update this string no matter what.

Reply:
I actually hadn't noticed this was one gigantic string rather than some three to five separate ones. For the translators it might be nicer to split them up a bit more in case of future updates.

Reply:
@Frenzie Took a bit but done, see update. Hopefully resolves your concerns and doesn't create too many new ones 😂